templates/waggle-dance-federation-glue.yml.tmpl

```diff
@@ -1,7 +1,7 @@
 %{if metastore_enabled~}
-  - name: ${prefix}
+  - name: ${ name == "" ? "prefix" : "name" }
     access-control-type: ${ writable_whitelist == "" ? "READ_ONLY" : "READ_AND_WRITE_ON_DATABASE_WHITELIST" }
-    database-prefix: ${prefix}_
+    database-prefix: '${ prefix == "" ? "" : "prefix_" }'
```

Comment:
Is the prefix_ resolved to `foo_` if prefix is set to foo? This looks weird, but maybe I don't speak enough yaml template

Reply:
Tested, yeah a bit strange also to me, but probably terraform does just string replacement